packages/mocks/src/index.ts

```diff
@@ -1 +1,3 @@
-export * as consoleHandlers from './consoleHandlers';
+// @index(['./**/*.ts', './**/*.tsx', /__/g, /index.ts/g], f => `export * from '${f.path}';`)
```

Comment:
interesting syntax:) what does it do?

Reply:
ok. so it's some IDE specific extension. I'd rather avoid it if we want to have stable CI runs.

Reply:
it's a human readable comment, why would it impact ci ?
for users that don't use generate index, it's a hint what files the should include
for users of generate index it automatically include this files

Reply:
in this case it say, if you add a file to this 'index.ts' file, please add .ts .tsx files and omit files that have __ in there paths, e.g. __test__ and index other files, we can make the hint more expliciit, e.g. __tests__ instead of __, but it's very helpful

Reply:
it seems CI and IDE will see different list of exports

Reply:
no, it's just a comment, it does not change how IDE see things.

Reply:
the way it works is:
a. you create a comment with the hints, very js syntax like, so it's very human readable
b. nothing happen
c. you manually run the generate index command
d. the generate-index package will run on this file, and generate the index according to the hints in the comment.
e. now you can look at the code that was generated and save or not save it